Packs/CortexExposureManagement/Playbooks/Cortex_EM_-_Exposure_Issue.yml

```diff
@@ -0,0 +1,736 @@
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Sleep for X seconds.
+      id: 4847a30a-8a30-4980-8195-1cd2ea603a0e
+      iscommand: false
+      name: Sleep for 1 hour
+      script: Sleep
+      type: regular
+      version: -1
+    taskid: 4847a30a-8a30-4980-8195-1cd2ea603a0e
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 840,
+          "y": 470
+        }
+      }
+  "10":
+    continueonerrortype: ""
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "11"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      asset_id:
+        simple: ${alert.asset_ids}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Cortex Core - IR
+      description: Get UAI asset info.
+      id: f3dcbcf4-47b0-4d59-8314-e5fd63a0c6f2
+      iscommand: true
+      name: core-get-asset-details
+      script: Cortex Core - IR|||core-get-asset-details
+      type: regular
+      version: -1
+    taskid: f3dcbcf4-47b0-4d59-8314-e5fd63a0c6f2
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 840,
+          "y": 630
+        }
+      }
+  "11":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: Core.CoreAsset
+          operator: isExists
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "11"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "15"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Checks if an asset was returned from the last command.
+      id: c4029b72-15e0-4e0e-891a-9fec5192b346
+      iscommand: false
+      name: Was an asset found?
+      type: condition
+      version: -1
+    taskid: c4029b72-15e0-4e0e-891a-9fec5192b346
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 840,
+          "y": 790
+        }
+      }
+  "12":
+    continueonerrortype: ""
+    id: "12"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      system_ids:
+        simple: ${Core.CoreAsset.xdm__asset__name}
+      ownerrelatedfield:
+        simple: xdmremediationowners
+      owners:
+        simple: ${vmassetownerunrankedraw}
+      tenantcommand:
+        simple: "True"
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Recommend most likely remediation owners from those surfaced by other enrichments.
```

Comment:
This script determines the most likely remediation owners from those surfaced by other enrichments.